src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
     public String getRandomColor() {
-        return null;
+        int randomNumberColor = random.nextInt(colors.length);
+        return colors[randomNumberColor].toString();
```

Comment:
As described in [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants), let's use name() method here instead of toString()